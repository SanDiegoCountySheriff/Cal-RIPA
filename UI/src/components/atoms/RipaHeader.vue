<template>
  <div class="ripa-header">
    <span class="tw-text-xl tw-uppercase">{{ value }}</span>
  </div>
</template>

<script>
export default {
  name: 'ripa-header',

  props: {
    value: {
      type: String,
      default: '',
    },
  },
}
</script>
