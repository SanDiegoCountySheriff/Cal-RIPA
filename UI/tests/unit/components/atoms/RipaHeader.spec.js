import RipaHeader from '@/components/atoms/RipaHeader.vue'
import { shallowMount } from '@vue/test-utils'
import Vuetify from 'vuetify'

describe('Ripa Header', () => {
  let vuetify
  let wrapper = null

  beforeEach(() => {
    vuetify = new Vuetify()
  })

  const factory = propsData => {
    return shallowMount(RipaHeader, {
      vuetify,
      propsData: {
        ...propsData,
      },
    })
  }

  it('should match snapshot', () => {
    wrapper = factory()
    expect(wrapper.element).toMatchSnapshot()
  })

  it('should display value', () => {
    wrapper = factory({ value: 'Value' })
    expect(wrapper.html()).toContain('Value')
  })
})
