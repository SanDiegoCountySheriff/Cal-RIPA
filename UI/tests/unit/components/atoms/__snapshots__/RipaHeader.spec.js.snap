// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ripa Header should match snapshot 1`] = `
<div
  class="ripa-header"
>
  <span
    class="tw-text-base md:tw-text-lg tw-uppercase"
  >
    
  </span>
</div>
`;
